packages/venia-ui/lib/util/makeUrl.js

```diff
@@ -44,8 +44,14 @@ const mediaBases = new Map()
  * @param {string} props.type - "image-product" or "image-category"
  * @param {number} props.width - the desired resize width of the image
  * @param {number} props.height - the desired resize height of the image
+ * @param {number} props.quality - the desired quality of the image
+ * @param {bool} props.crop - should the image be cropped
+ * @param {string} props.fit - how should the image be fit with the dimensions: bounds, cover, crop
```

Comment:
express-sharp only seems to assign values of `cover` or `outside` to `fit`, and assigns that value solely based on the value of `crop`.  https://github.com/magento-research/express-sharp/blob/develop/lib/transform.js#L23

Reply:
This is due to Fastly: https://docs.fastly.com/api/imageopto/fit